packages/react-native/src/utils/versions.ts

```diff
@@ -1,30 +1,27 @@
 export const nxVersion = require('../../package.json').version;
 
-export const reactNativeVersion = '~0.76.3';
+export const reactNativeVersion = '~0.79.3';
 
 export const typesNodeVersion = '18.16.9';
 
-export const reactNativeCommunityCliVersion = '~15.0.1';
-export const reactNativeCommunityCliPlatformAndroidVersion = '~15.0.1';
-export const reactNativeCommunityCliPlatformIosVersion = '~15.0.1';
+export const reactNativeCommunityCliVersion = '~18.0.0';
+export const reactNativeCommunityCliPlatformAndroidVersion = '~18.0.0';
+export const reactNativeCommunityCliPlatformIosVersion = '~18.0.0';
 
-export const reactNativeBabelPresetVersion = '~0.76.3';
-export const reactNativeMetroConfigVersion = '~0.76.3';
-export const reactNativeWebVersion = '~0.19.13';
+export const reactNativeBabelPresetVersion = '~0.79.3';
+export const reactNativeMetroConfigVersion = '~0.79.3';
+export const reactNativeWebVersion = '~0.20.0';
 
-export const reactVersion = '~18.3.1';
-export const reactDomVersion = '~18.3.1';
-export const reactTestRendererVersion = '~18.3.1';
-export const typesReactVersion = '~18.3.12';
+export const reactVersion = '19.0.0';
+export const reactDomVersion = '19.0.0';
+export const reactTestRendererVersion = '19.0.0';
```

Comment:
I think react-test-renderer is deprecated in the new react:
https://react.dev/blog/2024/04/25/react-19-upgrade-guide#deprecated-react-test-renderer